bears/vcs/git/CISkipInspectBear.py

```diff
@@ -0,0 +1,63 @@
+import re
+
+from bears.vcs.git.GitCommitMetadataBear import GitCommitMetadataBear
+from coalib.bears.GlobalBear import GlobalBear
+from coalib.results.Result import Result
+
+
+class CISkipInspectBear(GlobalBear):
+    LANGUAGES = {'Git'}
+    AUTHORS = {'The coala developers'}
+    AUTHORS_EMAILS = {'coala-devel@googlegroups.com'}
+    LICENSE = 'AGPL-3.0'
+
+    BEAR_DEPS = {GitCommitMetadataBear}
+
+    def run(self, dependency_results, filename_patterns: list,
+            appveyor_ci: bool = False, **kwargs):
+        """
+        Check the HEAD commit to see if it disables CI build and
+        return result accordingly. Supported CI include AppVeyor,
+        Bitrise, Circle CI, GitLab CI, Scrutinizer, Semaphore,
+        Shippable, Travis CI and wercker.
+
+        :param filename_patterns:   A list of regex to identify files
+                                    that should not skip CI builds.
+        :appveyor_ci:               Whether AppVeyor is used by the
+                                    project or not.
+        """
+        for result in dependency_results[GitCommitMetadataBear.name]:
+
+            supported_regex = r'\[ci skip\]|\[skip ci\]'
+
+            # AppVeyor allows to skip build by commit if the
+            # supported sequence is present in the title of the
+            # commit only. All other CI tools allow these sequences
+            # to be present anywhere in the commit message.
+            if appveyor_ci:
+                supported_regex = r'\[ci skip\]|\[skip ci\]|\[skip appveyor\]'
```

Comment:
I think we might want to directly append `|\[skip appveyor\]` to `supported_regex` instead creating a whole different variable